packages/itwinui-css/backstop/tests/index.html

```diff
@@ -71,7 +71,7 @@
       @import url("@itwin/itwinui-variables");
       @import url("@itwin/itwinui-variables/os.css");
       @import url("@itwin/itwinui-css/global");
```

Comment:
`global` is not needed if we have `all`

Reply:
Removed the import of `global.css`. Also removed watching for changes in global.scss in the `build:watch:scss` script since there is no import of `global.css`. [`7b00ed5`](https://github.com/iTwin/iTwinUI/pull/861/commits/7b00ed5f4679fb0f1a357421dcaaf20b59df8bcb)